lib/main.dart

```diff
@@ -48,12 +48,6 @@ class GalleryApp extends StatelessWidget {
         builder: (context) {
           final options = GalleryOptions.of(context);
           return MaterialApp(
-            // By default on desktop, scrollbars are applied by the
-            // ScrollBehavior. This overrides that. All vertical scrollables in
-            // the gallery need to be audited before enabling this feature,
-            // see https://github.com/flutter/gallery/issues/523
-            scrollBehavior:
-                const MaterialScrollBehavior().copyWith(scrollbars: false),
```

Comment:
I think the error presented itself in other demo apps in the gallery as well. We recently changed the default PrimaryScrollController behavior (https://github.com/flutter/flutter/pull/102099) to better avoid that error in general, so it may be ok.

I think the Shrine app was one that also had this issue.

Reply:
Ah I see. Can't repro on Shrine anymore.